external/external_custom/dockerfile/test.sh

```diff
@@ -0,0 +1,53 @@
+ #/bin/bash
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+source $(dirname "$0")/test_base_functions.sh
+
+# Set up Java to be used by the external_custom test
+
+if [ -d /java/jre/bin ];then
+	echo "Using mounted Java8"
+	export JAVA_BIN=/java/jre/bin
+	export JAVA_HOME=/java
+	export PATH=$JAVA_BIN:$PATH
+elif [ -d /java/bin ]; then
+	echo "Using mounted Java"
+	export JAVA_BIN=/java/bin
+	export JAVA_HOME=/java
+	export PATH=$JAVA_BIN:$PATH
+else
+	echo "Using docker image default Java"
+	java_path=$(type -p java)
+	suffix="/java"
+	java_root=${java_path%$suffix}
+	export JAVA_BIN="$java_root"
+	$JAVA_BIN/java -version
+	export JAVA_HOME="${java_root%/bin}"
+fi
+
+echo_setup
+
+# export MAVEN_OPTS="-Xmx1g"
+
+cd /flink
```

Comment:
this is specific to repo that was cloned, so should not hardcode to flink

Reply:
We should use another variable for this and for getting the value for this can we set up a variable DIR in the test.properties file and read from it?

Reply:
this will always be the name of the external test repo that we cloned, so we can figure it out based on the value of that variable (which I think you are currently calling EXTERNAL_CUSTOM_TARGET but really that is a repo_url... probably be clearer to call that EXTERNAL_CUSTOM_REPO

so, for https://github.com/apache/camel-quarkus.git, cd /[camel-quarkus](https://github.com/adoptium/aqa-tests/blob/master/external/camel/dockerfile/camel-test.sh#L45)  and so forth

Reply:
I think we can just reuse the cmd to get folder name from git URL
https://github.com/adoptium/aqa-tests/blob/14f5a8348e22e5616dea2ea001ab8d693cddfeee/external/dockerfile_functions.sh#L450

Reply:
@smlambert Yes, I will change the name of the variable @lxia Yes we should get the command from the git URL. I will make the changes. Thanks!